### DISCUSSION TITLE
General blog on kwok

### DISCUSSION THREAD

**Discussion by ashok-an** [kubernetes-sigs/kwok#466] [Show and tell]
I was exploring kwok as a replacement for kind (which I realize it is not), but ended up blogging my experience so far with it - https://dev.to/ashokan/kwok-kubernetes-without-kubelet-ie1

Is my understanding correct? any feedback

**Comment by wzshiming** [comment]
Amazing blog you are correct in your understanding, maybe the lack of clear comparisons in the documentation caused your previous misunderstanding, should we add a separate page comparison for KWOK and other tools?

**Comment by ashok-an** [comment]
YES please

-- Ashok
